docs/docs/add-custom-webpack-config.md

```diff
@@ -25,9 +25,9 @@ call each build type a "stage". The following stages exist:
 4. build-html: production build static HTML pages
 5. build-javascript: production JavaScript build. Creates route bundles as well as a `commons` and `app bundle`.
 
-Check [https://github.com/gatsbyjs/gatsby/blob/1.0/packages/gatsby/src/utils/webpack.config.js](https://github.com/gatsbyjs/gatsby/blob/1.0/packages/gatsby/src/utils/webpack.config.js) for the source.
+Check [https://github.com/gatsbyjs/gatsby/blob/1.0/packages/gatsby/src/utils/webpack.config.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/webpack.config.js) for the source.
 
-There are many plugins in the Gatsby repo using this API to look to for examples e.g. [Sass](/docs/packages/gatsby-plugin-sass/), [Typescript](/docs/packages/gatsby-plugin-typescript/), [Glamor](/docs/packages/gatsby-plugin-glamor/), and many more!
+There are many plugins in the Gatsby repo using this API to look to for examples e.g. [Sass](/packages/gatsby-plugin-sass), [Typescript](/packages/gatsby-plugin-typescript/), [Glamor](/packages/gatsby-plugin-glamor/), and many more!
```

Comment:
These links are for the website not for github necessarily... hmmm I guess we could change the package links there though so they're the same. 

E.g. this is a package link on the site https://www.gatsbyjs.org/docs/packages/gatsby-plugin-glamor/